testing/tests/calls.rs

```diff
@@ -106,6 +106,73 @@ mod test_double_attr_arg {
     }
 }
 
+#[test]
+fn test_caller() {
+    #[derive(Template)]
+    #[template(
+        source = r#"
+{%- macro test() -%}
+{{- caller() -}}
+{%- endmacro -%}
+{%- call() test() -%}
+nested
+{%- endcall -%}
+"#,
+        ext = "html"
+    )]
+    struct CallerEmpty {}
+    let x = CallerEmpty {};
+    assert_eq!(x.render().unwrap(), "nested");
```

Comment:
Can you add the following test to ensure backlines are not removed around "nested" please?

```jinja
{%- macro test() -%}
{{- caller() -}}
{%- endmacro -%}
{%- call() test() %}
nested
{% endcall -%}
```

Reply:
is this expected??

```
\nnested\n\n
```

Reply:
No it's not.

Reply:
umm i made a mistake somewhere :?

Reply:
Likely. Good Luck with whitespace handling. :wink: 

If you can't figure it out, I can take a look later on.

Reply:
yea ... can't quite figure it out

Reply:
Just came back from conference. Give me some (random amount of) time to take a look at it and try to figure out what's wrong.

Reply:
Thanks for letting me know. Conference? Sounds like fun did you present anything.

Reply:
Nah, it was rustweek/rust all-hands. It was time to meet with my teams mates and to talk with a loooot of people. Social battery is empty now. :')